trace/cloud-trace-demo-app/app/app.py

```diff
@@ -32,6 +33,7 @@
 propagator = google_cloud_format.GoogleCloudFormatPropagator()
```

Comment:
in the docs, this line is included in the region tag. Should it be? if so, adjust the region tag to encompass it

Reply:
Done